ex11.py

```diff
@@ -0,0 +1,13 @@
+print "How old are you?",
+age = raw_input()
```

Comment:
raw_input()
- When to get input from user.
- To prompt to the user to provide input.
- To read a string from standard input.